python/overrides.js

```diff
@@ -12,6 +12,7 @@
     // Failing on the universal server 1.10 and lower
     // check on new universal server version
     "check window on mobile web android": {skip: true},
-    'should waitBeforeCapture in open': { skipEmit: true },
-    'should waitBeforeCapture in check': { skipEmit: true },
+
+    // TODO verify and enable
+    'should send agentRunId': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8370302)
```suggestion
    "should send agentRunId": {skipEmit: true},
```